libraries/rtos/rtx/TARGET_CORTEX_M/cmsis_os.h

```diff
@@ -121,7 +121,11 @@ used throughout the whole project.
 #    define WORDS_STACK_SIZE   128
 #endif
 
-#define DEFAULT_STACK_SIZE         (WORDS_STACK_SIZE*4)
+#if defined(TARGET_STM32L053R8) || defined(TARGET_STM32L053C8)
```

Comment:
Why this is needed ? I found DEFAULT_STACK_SIZE used only in the Thread ctor, which has it as default parameter. So those 2 targets should define that parameter when created a new thread.